README.md

```diff
@@ -1,8 +1,8 @@
 # MUD
 
 <div align="center">
-<img src="docs/public/logo512-black-w-background.png" width="200" style="margin: 0 0 30px 0;" alt="MUD logo" />
-<p>MUD - Engine for Autonomous Worlds</p>
+<img src="docs/public/logo512-black.svg" width="200" style="margin: 0 0 30px 0;" alt="MUD logo" />
```

Comment:
```suggestion
<img src="docs/public/mud-cover-photo.png" width="200" style="margin: 0 0 30px 0;" alt="MUD logo" />
```
how does the big and bold logo look?

Reply:
I think that mud-cover-photo.png would look great if we could have the orange background all around but with the orange background just on the image I don't like it. But I'm not great at aesthetics. We'll go with your proposal.